cli/pkg/gitutil/gitutil.go

```diff
@@ -132,3 +132,63 @@ func SplitGithubURL(githubURL string) (account, repo string, ok bool) {
 
 	return account, repo, true
 }
+
+type SyncStatus int
+
+const (
+	NA       SyncStatus = iota
+	MODIFIED            // Local branch has untracked/modified changes
+	AHEAD               // Local branch is ahead of remote branch
+	SYNCED              // Local branch is in sync with remote branch
+)
```

Comment:
Doesn't feel like an idiomatic Go enum:

1. Use "unspecified" instead of "NA"
2. Enums should be pascal case, not uppercase
3. They would also usually include the type name (before or after)

For example:

```
SyncStatusUnspecified
SyncStatusModified
SyncStatusAhead
SyncStatusSynced
```

(The Go authors recommendation is to have the type name after, if contextually relevant, but I always found that hard to navigate in autocomplete)

Reply:
1. Unknown/unspecified/NA all seem to specify the same meaning and the difference is usually subjective. 
3. I think Go authors are not consistent here. I get an impressions at multiple places that the general recommendation is to be as less verbose as possible. I personally have seen so many examples of enums not putting the type name either before or after. Example here : https://github.com/google/go-cloud/blob/04a775a88f1cb11f0d8d9131d1de9d5a6843b355/internal/gcerr/gcerr.go#L36
I would prefer it to be short.